crates/rune-modules/src/http.rs

```diff
@@ -69,26 +69,1359 @@ pub fn module(_stdio: bool) -> Result<Module, ContextError> {
     module.ty::<Response>()?;
     module.ty::<RequestBuilder>()?;
     module.ty::<StatusCode>()?;
+    module.ty::<Version>()?;
     module.ty::<Error>()?;
 
-    module.function_meta(Client::new)?;
     module.function_meta(get)?;
 
+    module.function_meta(Client::new)?;
     module.function_meta(Client::get)?;
     module.function_meta(Client::post)?;
+    module.function_meta(Client::put)?;
+    module.function_meta(Client::delete)?;
+    module.function_meta(Client::head)?;
 
     module.function_meta(Response::text)?;
     module.function_meta(Response::json)?;
     module.function_meta(Response::bytes)?;
     module.function_meta(Response::status)?;
+    module.function_meta(Response::version)?;
+    module.function_meta(Response::content_length)?;
+    // module.function_meta(Response::remote_addr)?; TODO: Make rune net module
 
     module.function_meta(RequestBuilder::send)?;
     module.function_meta(RequestBuilder::header)?;
-    module.function_meta(RequestBuilder::body_bytes)?;
+    module.function_meta(RequestBuilder::basic_auth)?;
+    module.function_meta(RequestBuilder::bearer_auth)?;
     module.function_meta(RequestBuilder::fetch_mode_no_cors)?;
+    module.function_meta(RequestBuilder::body_bytes)?;
+    // module.function_meta(RequestBuilder::form)?; TODO: Make with support anonymous objects: future work.
+    // module.function_meta(RequestBuilder::timeout)?; TODO: Make rune net module
+
+    module
+        .constant(
+            "CONTINUE",
+            StatusCode {
+                inner: reqwest::StatusCode::CONTINUE,
+            },
+        )
+        .build_associated::<StatusCode>()?
+        .docs(docstring! {
+            /// Status Code: Continue
+            ///
+            /// # Examples
+            ///
+            /// ```rune
+            /// use http::StatusCode;
+            ///
+            /// let status_code = StatusCode::CONTINUE;
+            /// ```
+        })?;
```

Comment:
The constants still won't work right since we don't have support for constructing constant values of external objects? So you'll have to remove them for now.